pages/ksphere/kommander/1.1/workspaces/index.md

```diff
@@ -16,7 +16,7 @@ For example, you can create separate workspaces for each department, product, or
 
```

Comment:
hm, don't we also have to rename all these folders from `1.1.0-beta` -> `1.1.0` ?